src/page-components/Home/styles.module.css

```diff
@@ -0,0 +1,141 @@
+.sm__handles {
+  top: 200px;
+  right: 35px;
+}
+
+.sm__handles ul {
+  gap: 25px;
+}
+/*
```

Comment:
Do we still need this commented code? If not we should remove them. If we do explain why.